xpdacq/xpdacq.py

```diff
@@ -422,6 +430,63 @@ def beamtime(self, bt_obj):
     def __call__(self, sample, plan, subs=None, *,
                  verify_write=False, dark_strategy=periodic_dark,
                  raise_if_interrupted=False, **metadata_kw):
+        """
+        Execute a plan
+
+        Any keyword arguments other than those listed below will
+        be interpreted as metadata and recorded with the run.
+
+        Parameters
+        ----------
+        sample : int or dict-like
+            Sample metadata. If a beamtime object is linked,
+            an integer will be interpreted as the index appears in the
+            ``bt.list()`` method, corresponding metadata will be passed.
+            A customized dict can also be passed as the sample
+            metadata.
+        plan : int or generator
+            Scan plan. If a beamtime object is linked, an integer
+            will be interpreted as the index appears in the
+            ``bt.list()`` method, corresponding scan plan will be
+            A generator or that yields ``Msg`` objects (or an iterable
+            that returns such a generator) can also be passed.
+        subs: callable, list, or dict, optional
+            Temporary subscriptions (a.k.a. callbacks) to be used on
+            this run. Default to None. For convenience, any of the
+            following are accepted:
+
+            * a callable, which will be subscribed to 'all'
+            * a list of callables, which again will be subscribed to 'all'
+            * a dictionary, mapping specific subscriptions to callables or
+              lists of callables; valid keys are {'all', 'start', 'stop',
+              'event', 'descriptor'}
+
```

Comment:
extra space?

Reply:
This extra space is needed, for the list above
 (most of the doc is copied from ``bluesky`` doc)